Assets/MRTK/Examples/packagetemplate.json

```diff
@@ -25,7 +25,8 @@
       },
     "dependencies": {
         "com.microsoft.mixedreality.toolkit.foundation": "%version%",
-        "com.microsoft.mixedreality.toolkit.extensions": "%version%"
+        "com.microsoft.mixedreality.toolkit.extensions": "%version%",
+        "com.unity.textmeshpro": "2.1.1"
```

Comment:
@davidkline-ms a couple of questions:

1. Should we use version 1.4.1 here like how we have that version here: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Packages/manifest.json#L4
2. Should this be another templated version as well, so that the version that's used here is consistent with the version here: https://github.com/microsoft/MixedRealityToolkit-Unity/blob/mrtk_development/Packages/manifest.json#L4. Otherwise going forward these two version dependencies could get inconsistent

Reply:
I should note that we can separate 1 and 2 - I think unblocking things/making things work is top priority, fixing the long term integration can be done in a separate PR by us.

Reply:
re 1. you require Unity 2019.4 and the 'verified' version of TMP is 2.1.1, so that's what I went with.